proto/osmosis/poolmanager/v1beta1/tx.proto

```diff
@@ -37,6 +37,27 @@ message MsgSwapExactAmountInResponse {
   ];
 }
 
+// ===================== MsgSplitRouteSwapExactAmountIn
+message MsgSplitRouteSwapExactAmountIn {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  repeated SwapAmountInSplitRoute routes = 2 [ (gogoproto.nullable) = false ];
+  string token_in_denom = 3
+      [ (gogoproto.moretags) = "yaml:\"token_in_denom\"" ];
+  string token_out_min_amount = 4 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"token_out_min_amount\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+message MsgSplitRouteSwapExactAmountInResponse {
+  string token_out_amount = 1 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"token_out_amount\"",
+    (gogoproto.nullable) = false
+  ];
```

Comment:
Would it at all be useful to return a coin instead of just the int? I know it can probably get pulled from the pool itself, but seems like this would be nice as one less step for integrators. Feel free to resolve if disagree, no strong opinion.

Reply:
The denom is provided by the client as part of the routes so I don't think it's necessary